tests/test_api.py

```diff
@@ -832,6 +832,55 @@ def test_get_collection_items(config, api_):
     assert code == 400
 
 
+def test_get_collection_items_postgresql_cql():
+    """
+    Test for PostgreSQL CQL - requires local PostgreSQL with appropriate
+    data.  See providers/postgresql.py for details.
+
+    First test good request, then error conditions.
```

Comment:
What do you think about separating tests on the different conditions?

Reply:
Separated by @ximenesuk.